### DISCUSSION TITLE
Release 3.4.7

### DISCUSSION THREAD

**Discussion by Fruchtzwerg94** [vvvv/SVG#1147] [General]
Hi,

since there are again some fixes and changes pending, I want to ask what are the plans for the next release. I would love to see the changes in some of my projects like [PlantUmlViewer](https://github.com/Fruchtzwerg94/PlantUmlViewer).
Any ideas / plans when the version 3.4.7 will be release officially?

BR, Philipp

**Comment by mrbean-bremen** [comment]
We do patch releases on demand, so I think we can do one soon. We probably want to merge one open PR first which is basically ready, if @paulushub agrees, and make a release - so probably in a couple of days.

**Reply by Fruchtzwerg94** [reply]
Great, I am looking forward to it. Thanks!

**Reply by mrbean-bremen** [reply]
The release [is out](https://www.nuget.org/packages/Svg/3.4.7).
